src/App.tsx

```diff
@@ -1,204 +1,45 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useState, useEffect, useContext } from 'react';
+import { getTodos } from './api/todos';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { Main } from './components/Main';
+import {
+  ErrorContext,
+  LoadingContext,
+  TodosContext,
+} from './components/TodoContext';
+import { Status } from './types/Status';
+import { ErrorNotification } from './components/ErrorNotification';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const { list, setList } = useContext(TodosContext);
```

Comment:
- What `list` is this?
- Create a hooks to avoid passing context every time: `const useTodos = () => useContext(TodosContext);`

Reply:
todo list

Reply:
> todo list

If it's list of todos so name it like this or just `todos` so everyone can understand